### ISSUE TITLE
mail.js does not work with exim mail servers :(

### ISSUE THREAD

**Issue by xdascx** [totaljs/framework#297]
I have tried debugging the issue but have given up because I needed to get other things working.

issue:-
1) 503 AUTH COMMAND USED WHEN NOT ADVERTISED 
2) 550 PERMITTED TO RELAY THROUGH THIS SERVER WITHOUT AUTHENTICATION
1 and 2 - resolve by pushing ("EHLO") before "AUTH PLAIN" and also changing "AUTH PLAIN" to "AUTH PLAIN LOGIN"

3) need to add "HELO" in front "MAIL" 

4) 503 SENDER NOT YET GIVEN Error: 503 SENDER NOT YET GIVEN

basically it's just a matter of hooking up different commands to send mail for exim servers

**Comment by petersirka** [comment]
Hi @xdascx,
can you send me your SMTP informations on my e-mail? I must test it.

Thanks.

**Comment by xdascx** [comment]
Hi Peter,

Please use the following for testing, I'll be deleting the email account as soon as you're done with it. :)

username: tester@3byte.net
password: q-ymv7^KKyLs

Secure settings:-
moore.uk-noc.com
SMTP Port: 465

Insecure settings:-
mail.3byte.net
SMTP Port: 26

**Comment by petersirka** [comment]
Hi @xdascx,
try to change your exim mail server:

> Exim does have a work around for this, using allow_auth_unadvertised as described here, you can do something like this:

```
hosts   = *
control = allow_auth_unadvertised
```
__Source:__ <http://serverfault.com/questions/313562/exim-configuration-503-auth-command-used-when-not-advertised>. I will update code of the mail if your problem will persist.

Give me a feedback ASAP.

**Comment by xdascx** [comment]
Hi Peter,

Unfortunately the mail server is hosted by a 3rd party and I don't have access to any of the mail configurations :(

**Comment by petersirka** [comment]
Hi @xdascx,
I have fixed it. Please download the latest beta version (is stable) `npm install total.js@beta`.

Thanks!

**Comment by xdascx** [comment]
Brilliant!  Works like a charm! :D
